spark/ingestion/src/main/scala/feast/ingestion/StreamingPipeline.scala

```diff
@@ -124,6 +148,19 @@ object StreamingPipeline extends BasePipeline with Serializable {
           .option("entity_repartition", "false")
           .save()
 
+        config.source match {
+          case _: KafkaSource =>
+            Option(
+              rowsAfterValidation
+                .agg(max("timestamp") as "latest_timestamp")
+                .collect()(0)
```

Comment:
what if rowsAfterValidation would be empty?

Reply:
Good catch, added validation